recipes/dwave-preprocessing/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "dwave-preprocessing" %}
+{% set version = "0.3.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/dwave-preprocessing-{{ version }}.tar.gz
+  sha256: f01f02ba5387117685afbf7d5c741a1a0ca78f99946fc24613cfac61bc85f694
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+  host:
+    - cython >=0.29.21,<3.0
+    - dimod ==0.10.9
```

Comment:
Hi @randomir quick question: This is quite tight and means that we need to get the older version of `dimod` onto conda-forge as well. Can I use a wider band here as well, or is that required?

Reply:
@arcondello, you just updated dwave-preprocessing, can we relax this requirement now?

Reply:
Basically any version after dwave-preprocessing has been decoupled from dimod would be great because anything before that would be slightly more difficult to bring to CF.

Reply:
@arcondello Gentle ping, when you have a minute.

Reply:
Ok seems like this is working with the relaxed version so I'll continue with this to unblock the other stuff.